dts/bindings/gpio/nordic,gpio-pins.yaml

```
@@ -2,13 +2,13 @@
 # SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
 
 description: |
-  This allows to define the pin of the GPIO to be owned by the domain.
-  It does not provide any specyfic driver.
-  Its function is to allow us to use two functionalities:
+  This binding allows to define the pin of the GPIO to be owned by the domain.
```

```suggestion
  This binding allows to designate a GPIO pin to the domain.
```
just a suggestion - please double-check the meaning

---

Used `match` instead of `designate`. @nvlsianpu , please double-check.